src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -60,23 +87,47 @@ public DriveSubsystem() {
     m_rightFollowerMotor.follow(m_rightLeaderMotor);
 
     m_drive = new DifferentialDrive(m_leftLeaderMotor, m_rightLeaderMotor);
+    
+    // This code limits the motor current to 40 amps to prevent overheating
+
+    int rpm=3000;
+    int peakCurrent=70;
+    int continuousCurrent=40;
```

Comment:
We should move these current limits to the `Constants.java` file, probably under `DriveConstants`

Reply:
Ok i moved it. Had to change it to DriveConstants.peakCurrent,DriveConstants.continuousCurrent,DriveConstants.rpm to avoid errors.